### ISSUE TITLE
Mass connector updates to unpin CDK version

### ISSUE THREAD

**Issue by alafanechere** [airbytehq/airbyte#40523]
We want to do a best effort migration of all our connectors to CDK >1.
We should:
1. Run an airbyte-ci command which would set the CDK version to `<3` 
`airbyte-ci connectors --language=low-code --language=python upgrade-cdk "<3"` (This command has not been used so far for python connectors, some minor tweaks might be required).
2. Run `airbyte-ci connectors --modified bump-version patch 'update CDK to <3' 
3. Run `airbyte-ci connectors --modified pull-request`
4. Manually set the `auto-merge` label on created PRs.

## Open question

Shall we consecutively open two batches: 
* One batch setting version to `<2` to maximize the chances to get connectors to > 1
* A seconde batch to `<3`

**Comment by girarda** [comment]
@alafanechere is there no way to automatically set `auto-merge` on the PRs?

**Comment by natikgadzhi** [comment]
@girarda nothing stops us from slapping `auto-merge` labels on all of the generated PRs. `airbyte-ci` has all the knobs to implement the command — do you think Python sources can pick this up pretty please?

**Comment by girarda** [comment]
Yes, we can do this as part of the mass connector upgrades we'll take on to enable RFR, sensible default error handling, and consistent stream status messages
